### PR TITLE
Tests for TPCDS queries join ordering (v3)

### PR DESCRIPTION
**Pull request by kokosing**
Prefer toDataProvider instead of toArgumentsArrays

### CODE REVIEW

**Comment by kokosing** [comment]
Supersedes #29

File: presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/BaseJoinReorderingTest.java [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+            return visitPlan(node, indent + 1);
+        }
+
+        @Override
+        public Void visitTableScan(TableScanNode node, Integer indent)
+        {
+            stringBuilder.append(indentString(indent))
+                    .append(node.getTable().getConnectorHandle().toString())
+                    .append("\n");
+            return visitPlan(node, indent + 1);
```
Comment (losipiuk) [inline presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/BaseJoinReorderingTest.java]: no need for that, right?

**Review by losipiuk — COMMENTED** [review]
Buu - why merged. No green tick :/

**Comment by kokosing** [comment]
See http://jenkins.aws.starburstdata.com/job/presto-public-travis-build-pipeline/729/display/redirect

**Pull request merged**